### ISSUE TITLE
Natively add and show completion status and favourite icons in every game list

### ISSUE THREAD

**Issue by WildGenius** [JosefNemec/Playnite#2771]
### Feature description

Currently, there are a few themes that implement showing game completion status and favourite status as an icon in the games list (see [example from eMixedNite](https://raw.githubusercontent.com/eminaguil/eMixedNite/master/screenshots/screenshot_02.png)). This should ideally be theme-agnostic, as it is adding visibility and usability to one of the best features of Playnite.

**Comment by JosefNemec** [comment]
Duplicate #1683

**Comment by darklinkpower** [comment]
> Duplicate #1683

I think this refers to something else. It's from this 

[Themes with completion status display as an icon in games list?](https://www.reddit.com/r/playnite/comments/sayo2z/themes_with_completion_status_display_as_an_icon/)

So from what I understand @WildGenius wants to display aditional information, including assign icons (Which would probably link it to https://github.com/JosefNemec/Playnite/issues/1683, although not the same thing) in the Details View Library game items

![image](https://user-images.githubusercontent.com/1389286/151285747-9b5efb70-12ae-402e-a858-22b1b44f4ee0.png)

**Comment by WildGenius** [comment]
More accurately, I wanted these icons. I did copy over this OP to the linked issue, so if the maintainer wants to track it more generally, I'm absolutely fine with that - after all, these are all set from fields.
![image](https://user-images.githubusercontent.com/8071762/151326051-f33ae2ec-73c4-4385-9535-527d7da7380d.png)

**Comment by JosefNemec** [comment]
So is this about adding additional game information/icons to entries in the game list?

**Comment by WildGenius** [comment]
Yes?.. Sorry, not sure what you're asking - the title and OP both explicitly mention the game list. I guess I should have included the annotated screenshot from the outset?..

**Comment by shakeyourbunny** [comment]
Please no. And if it will be implemented, make it OPTIONAL.

**Comment by WildGenius** [comment]
It is implemented in the linked theme such that you may have no extra information, icons, or both icons, last played, and gametime. See images. Out of curiosity, why "please no"? If you use the features, surely you want to easily see the games' status without going through the filters each time, and if you don't, there won't be any extra information to show.
![image](https://user-images.githubusercontent.com/8071762/151675672-0b716f5b-bd3b-4132-83c7-14adaaa82557.png)
![image](https://user-images.githubusercontent.com/8071762/151675690-78008380-07fd-4e66-bfd1-2c08b11840ce.png)
![image](https://user-images.githubusercontent.com/8071762/151675700-193f78d8-b865-4bfc-bc13-48818cbede4e.png)

**Comment by shakeyourbunny** [comment]
I just want as much entries in the list as possible for quick navigation without hassle, also the game icon is such in a way configured that it is not taller than the text in my installation. 

[![Bild capturet9kbk.png auf abload.de](https://abload.de/img/capturet9kbk.png)](https://abload.de/image.php?img=capturet9kbk.png)

Just imagine you have about 15.000 .. 20.000 games in this list and you will really have performance problems if Playnite has to enumerate through every list entry refresh. This is already a performance problem - even with the default theme - in this scenario if you group by series in the listing and have ... thousands of series there present. 

I am aware that there are several themes with these additional information and this is usually the reason I revert back to another theme if it is not possible to disable it, so it also should be able to be disabled in the standard client.

**Comment by WildGenius** [comment]
Of course, there should be a minimalist option present, I certainly didn't mean to imply this should be forced on anyone - after all, playnite is quite customiseable. However, I do believe that the functionality is worth it, even at the cost of a performance hit that is anything less than debilitating: from your example, it would be quite a hassle to manually check which entry in which series you completed/abandoned/did not play, while a simple second inline icon would solve that immediately.

**Comment by JosefNemec** [comment]
> Just imagine you have about 15.000 .. 20.000 games in this list and you will really have performance problems if Playnite has to enumerate through every list entry refresh. 

Playnite never enumerates the entire list, only once on startup, and that's just underlining data, not UI data. Adding additional information to those list entries would have very minimal performance impact on performance, since only data for visible entries are loaded to the UI.

**Comment by shakeyourbunny** [comment]
> Playnite never enumerates the entire list, only once on startup, and that's just underlining data, not UI data. Adding additional information to those list entries would have minimal performance impact on performance, since only data for visible entries are loaded to the UI.

I don't want to derail this, but it is very noticabely a delay when I group by series in the details listing and edit series information (rename series, change series order etc).

Coming back, I'm not against such an option for more information in the listing and see the usefulness of it, but it should be configurable.

**Comment by JosefNemec** [comment]
@shakeyourbunny 
> I don't want to derail this, but it is very noticabely a delay when I group by series in the details listing and edit series information (rename series, change series order etc).

Please open separate issue for it, I will look into it. If possible attach copy of your library.
